ui/app/AppLayouts/Chat/views/CreateChatView.qml

```diff
@@ -81,6 +91,22 @@ Page {
                     if (visible)
                         edit.forceActiveFocus()
                 }
+
+                onEnterKeyPressed: {
+                    entryAccepted(contactsList.itemAtIndex(contactsList.currentIndex));
+                }
+
+                onUpKeyPressed: {
+                    contactsList.decrementCurrentIndex();
+                }
+
+                onDownKeyPressed: {
+                    contactsList.incrementCurrentIndex();
+                }
+
+                Component.onCompleted: {
+                    membersSelector.entryAccepted(d.ownContact);
```

Comment:
This is so hacky.. :eyes: The proper solution would be to extend the model with ourselves rather than artificially accepting entry.

Reply:
so hacky? I don't see any "hack" on adding an entry manually when opening the view, you say to actually add to the contacts model our own self and still call it contacts model? what about the other places that model is used? this will be "hacky" then to have to exclude the first own entry from all other places it's used.

Reply:
I am not saying to add it to the global contacts model, just to the model used here.

Reply:
See: https://github.com/status-im/status-desktop/pull/7990